apps/studio/src/components/common/ContextMenu.vue.orig

```diff
@@ -0,0 +1,207 @@
+<template>
+  <!-- Original file souce copyright John Datserakis https://github.com/johndatserakis/vue-simple-context-menu -->
```

Comment:
What is this file and why do we need it?

Reply:
We don't need it uggghhh. meld likes to create files when I do a merge and they got auto staged for some reason. Will remove